frontend/ui/screensaver.lua

```diff
@@ -58,16 +58,6 @@ if G_reader_settings:hasNot("screensaver_hide_fallback_msg") then
 end
 
 local Screensaver = {
-    screensaver_provider = {
-        gif  = true,
-        jpg  = true,
-        jpeg = true,
-        png  = true,
-        svg  = true,
-        tif  = true,
-        tiff = true,
-        webp = true,
```

Comment:
"webp" was added as supported by screensaver, but mostly nowhere else. I don't remember if there was a reason, if it does not work as well as all the other image formats for other stuff, or if I just forgot to add it to ImageWidget.
(It's rare, so I dont mind if it is removed from the now unique image_ext - but if it works, let's keep it.)

Reply:
webp was not supported by ImageWidget, that's why I'd removed it.
Now it is, works everywhere.